src/Illuminate/Database/Query/Builder.php

```diff
@@ -2160,7 +2160,7 @@ public function getCountForPagination($columns = ['*'])
         } elseif (! isset($results[0])) {
             return 0;
         } elseif (is_object($results[0])) {
-            return (int) $results[0]->aggregate;
+            return (int) ($results[0]->aggregate ?? $results[0]->AGGREGATE);
```

Comment:
This is a breaking change.

Reply:
I'm sorry but, why is this a breaking change?

Reply:
Right now I'd expect `0` if `$results[0]->aggregate` is `null`. With this change, if `$results[0]->AGGREGATE` is somehow truthy, the returned result will not be `0` anymore, hence a breaking change.

Reply:
So, a properly change would be like this:
`return (int) (isset($results[0]->aggregate) ? $results[0]->aggregate : $results[0]->AGGREGATE);`

if 'aggregate' is defined but null, the return would be 0, if even the attribute is not defined it'd try 'AGGREGATE'. Right?

Reply:
That's not what I mean. No matter how you modify it, this is still a breaking change.